robottelo/ui/login.py

```diff
@@ -45,7 +45,7 @@ def is_logged(self):
             for cookie in self.browser.get_cookies()
         )
         # construct the link to the Dashboard web page
-        url_root = settings.server.get_url() + '/dashboard'
+        url_root = settings.server.get_url() + '/about'
```

Comment:
yeah, that page is hidden for most user roles, but accessible for all of them :)